donut/modules/auth/helpers.py

```diff
@@ -60,7 +60,7 @@ def handle_forgotten_password(username, email):
     with flask.g.pymysql_db.cursor() as cursor:
         cursor.execute(query, [username])
         result = cursor.fetchone()
-
+    email = email.lower()
```

Comment:
If the email in the database is not lower-cased, I think we should still send to that email. Can we just make the comparison `email.lower() == result['email'].lower()` and send the email to `result['email']` instead?